res/openvpn-gui-res-ru.rc

```diff
@@ -154,27 +155,27 @@ CAPTION "Advanced"
 FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
 BEGIN
-    GROUPBOX "Configuration Files", 201, 6, 12, 235, 45
-    LTEXT "Folder:", ID_TXT_FOLDER, 17, 25, 32, 10
-    LTEXT "Extension:", ID_TXT_EXTENSION, 17, 40, 52, 10
+    GROUPBOX "Файлы конфигурации", 201, 6, 12, 235, 45
+    LTEXT "Папка:", ID_TXT_FOLDER, 17, 25, 32, 10
+    LTEXT "Расширение:", ID_TXT_EXTENSION, 17, 40, 52, 10
     EDITTEXT ID_EDT_CONFIG_DIR, 53, 23, 150, 12, ES_AUTOHSCROLL
     EDITTEXT ID_EDT_CONFIG_EXT, 53, 38, 25, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_CONFIG_DIR, 208, 23, 25, 12
 
-    GROUPBOX "Log Files", 202, 6, 62, 235, 30
-    LTEXT "Folder:", ID_TXT_FOLDER, 17, 74, 32, 10
+    GROUPBOX "Файлы журнала", 202, 6, 62, 235, 30
+    LTEXT "Папка:", ID_TXT_FOLDER, 17, 74, 32, 10
     EDITTEXT ID_EDT_LOG_DIR, 53, 72, 150, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_LOG_DIR, 208, 72, 25, 12
 
-    GROUPBOX "Script Timeout", 201, 6, 97, 235, 60
-    LTEXT "Preconnect script timeout:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 100, 10
-    LTEXT "Connect script timeout:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 90, 10
-    LTEXT "Disconnect script timeout:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 90, 10
+    GROUPBOX "Ограничение времени выполнения скриптов", 201, 6, 97, 235, 60
+    LTEXT "Тайм-аут Preconnect-скрипта:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 100, 10
+    LTEXT "Тайм-аут Connect-скрипта:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 90, 10
+    LTEXT "Тайм-аут Disconnect-скрипта:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 90, 10
     EDITTEXT ID_EDT_PRECONNECT_TIMEOUT, 103, 108, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
     EDITTEXT ID_EDT_CONNECT_TIMEOUT, 103, 123, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
     EDITTEXT ID_EDT_DISCONNECT_TIMEOUT, 103, 138, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
 
-    AUTOCHECKBOX "Service only", ID_CHK_SERVICE_ONLY, 6, 162, 100, 12
+    AUTOCHECKBOX "Всегда использовать сервис", ID_CHK_SERVICE_ONLY, 6, 162, 100, 12
```

Comment:
"Always use the service" is not appropriate here. In "service only" mode the GUI will only support starting and stopping the "automatic service" and no configs can be connected or disconnected using the GUI. Its an arcane feature, pretty useless in my view, especially since it won't even work unless the GUI is run as admin. We'll remove it as soon as we have a better way of controlling prestarted instances.

For translation, I think an equivalent of  "service only mode" should do.